src/en/proposals/errant-vox-species.md

```diff
@@ -0,0 +1,94 @@
+
+  - There should be some sort of warning or confirmation when selecting them, so players, especially new players, don't end up unwittingly picking a choice that will kill their character for the wrong click. 
+
+  - They breathe nitrogen, but oxygen hurts them badly (tentative number is, they crit in 2 minutes in normal air. Afaik this is actually more lenient than in Spess13)  - they will start with a large nitrogen tank and breathing mask equipped
+    - because taking their mask off is BAD, eating and drinking is challenging. They need a nitrogen room, and someone needs to have made that room
+    - possible alternative to a nitrogen room is the as-yet uncoded feature to hold their breath, taking asphyxiation damage instead of poison (at an even faster rate). They can regenerate that automatically (assuming they don't miscalculate and go crit)
+    - their emergency food and water could be replaced with injectable reagents
+  
+ - They can't be cloned
+ - They take longer to rot (2x? 3x?)
+ - They have 50% poison resistance (oxygen bypasses this. poison is generally not a very common damage type/not a very useful resistance to have, so I feel 50 is fine)
+ 
+ - They have their own blood type, and when that system is expanded, it should be incompatible with "normal blood"
+ - Possibly their brain, (it actually being a piece of tech), is automatically ready to be placed into a borg chassis
+
+  - They are small/unassuming/fragile.
+    - Possibly a dwarf-sized sprite? (when you sprite it)
+    - They take ~ +25% damage from all Bruise
+    - They get drunk more easily
+    - Maybe they are worse at pulling things?
+    - They are easier to shove
+    - I thought they would deal sub-par damage but there have been some fairly strong feelings on Discord that their claws should be vicious claws. Maybe an exception can be made here, if they take more brute damage they won't really have an advantage in a brawl anyway. Unarmed damage probably isn't really all that important in a real fight anyway)
+  
+- They should have some actual mechanic involving screams (any such ability should consider that the hotbar cooldown does not apply to emoted screams. They might need to be an actual ability that also does a scream, rather than apply on any scream)
+  - they could have the ability to discern the positions of screams within a range, even through walls
+  - they could discern some information from screams, such as the coarse Health state of the person? (Good, Okay, Poor, Bad, Danger)
+  - their screams could apply a ! popup on nearby people with no additional effect?
+
+- They have a weird dietary range.
+  - It has been suggested that they be able to butcher without any tools. Very messily.
+  - Their thirst can only be satiated with welding fuel?
+  - Water makes them drunk
+  - Maybe "normal food" is less satiating to them than the weird stuff (if that can be categorized in some way)
```

Comment:
This is a possibility, but the way we categorize food right now is utterly horrific (see: lizards diet)

I think diet might be a bit out of scope of this for now, we simply don't have the things we need to make anything too gimmicky work without it being a chore.

Reply:
Diet is indeed a wasteland but I think most of what I have suggested in the update can be done right now, and I would be willing to make the nutrient etc reagents be more specific (the big obstacle there is that every impacted food would have to have it's reagents readjusted)